set.mm

```diff
@@ -656235,6 +656235,13 @@ D Fn ( I ... ( M - 1 ) ) /\
       LEFBCDMNO $.
   $}
 
+  $( A function value is a subset of the union of the range.  (An artifact of
+     our function value definition, compare ~ elfvdm ).  TODO:  Revise
+     ~ elunirn2 ?  (Contributed by SN, 10-Jan-2024.) $)
```

Comment:
~elfvunirn is a generalization of ~elunirn2 (has a weaker antecedent). Therefore, I propose that ~elfvunirn should replace ~elunirn2 (used 3 times): The contributor of ~elfvunirn should be @tirix, the contributor of ~elunirn2 (which is older):

`(Contributed by Thierry Arnoux, 13-Nov-2016.)  (Revised by SN, 10-Jan-2024.)`

Reply:
I would still like  ~elunirn2 to be replaced by ~elfvunirn. Any objections?

Reply:
Done, although it seems ~ elrnust is an instance of ~ elfvunirn .

@tirix I revised ~ isanmbfm and now it doesn't use its first two hypotheses, can they be removed?

Reply:
Yes, sure, thanks!